cmd/kava/cmd/root.go

```diff
@@ -69,7 +71,12 @@ func NewRootCmd() *cobra.Command {
 
 			customAppTemplate, customAppConfig := servercfg.AppConfig("ukava")
 
-			return server.InterceptConfigsPreRunHandler(cmd, customAppTemplate, customAppConfig)
+			return server.InterceptConfigsPreRunHandler(
+				cmd,
+				customAppTemplate,
+				customAppConfig,
+				tmcfg.DefaultConfig(),
```

Comment:
Includes tendermint config option here, unsure if we want any modifications to it.

Reply:
I don't think we make any standard changes to the default file.